Dockerfile

```diff
@@ -15,7 +15,8 @@
 # Build the manager binary
 FROM golang:1.20 as builder
 
-ARG TARGETARCH=amd64
+ARG ARCH ?= $(shell go env GOARCH)
```

Comment:
If I understand right multi-arch builds are then done somewhere else - this change is just enabling them. 

Could we make the docker build in the Makefile multi-arch? Or is it better to just have that as a detail of the release process?

Reply:
@killianmuldoon I'll add a note into README to to build such images. I don't want to change make target to not complicate develpment process